.gitignore

```diff
@@ -336,3 +336,4 @@ ASALocalRun/
 
 # Ignore Mac DS_Store files
```

Comment:
is this code much different from https://github.com/dotnet/iot/tree/master/src/devices/Ssd1306 ? if they are similar might be worth to unify them into Ssd13XX or something similar

Reply:
I looked at this but felt that the differences between are fairly major i.e.

- Ssd1306 is currently I2C only and Ssd1351 is SPI only with no option for I2C.
- The way that commands are sent to the device is quite different. The Ssd1351 distinguishes between the command and following parameter bytes with the used of the DC pin whereas the Ssd1306 uses the DC ping to distinguish between commands and data.
- The commands are quite different and the way that they address data.

We could of course use flow control statements to cover both devices but I think that the differences are such that most of the complexity would be in the flow control statements and not in the code that actually controls the binding.

Personally I think that what needs to be done in the future is to come to a definitive decision about what interfaces etc. we expect display devices to support but I think other contributors will have a better insight into that then myself.